.changeset/bright-apples-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+"@shadeprotocol/shadejs": patch
+---
+
+Added stkd-scrt messages and services
```

Comment:
this was also adjustment of some dshd stuff right?

Reply:
just a refactor of the name

Reply:
please include that in the changeset for clarity